sdk/core/core/src/az_http_private.h

```diff
@@ -27,19 +27,21 @@
 AZ_NODISCARD AZ_INLINE az_result _az_http_request_mark_retry_headers_start(_az_http_request* p_hrb)
 {
   AZ_PRECONDITION_NOT_NULL(p_hrb);
-  p_hrb->_internal.retry_headers_start_byte_offset = p_hrb->_internal.headers_length * (int32_t)sizeof(az_pair);
+  p_hrb->_internal.retry_headers_start_byte_offset
+      = p_hrb->_internal.headers_length * (int32_t)sizeof(az_pair);
   return AZ_OK;
 }
 
 AZ_NODISCARD AZ_INLINE az_result _az_http_request_remove_retry_headers(_az_http_request* p_hrb)
 {
   AZ_PRECONDITION_NOT_NULL(p_hrb);
-  p_hrb->_internal.headers_length = p_hrb->_internal.retry_headers_start_byte_offset / (int32_t)sizeof(az_pair);
+  p_hrb->_internal.headers_length
+      = p_hrb->_internal.retry_headers_start_byte_offset / (int32_t)sizeof(az_pair);
   return AZ_OK;
 }
 
 /**
- * @brief sets buffer and parser to its initial state
+ * @brief Sets buffer and parser to its initial state.
```

Comment:
This should be the following. Don't mention internal implementation detail as part of public facing docs.
```suggestion
 * @brief Sets the \p http_response to its initial state.
```

Reply:
well, this is not public facing. This is private header, only visible to developers